### DISCUSSION TITLE
How to I use the Windows Features?

### DISCUSSION THREAD

**Discussion by BrandonWanHuanSheng** [microsoft/winget-pkgs#107536] [Q&A] [answered]
How to I use this?
  Dependencies:
    WindowsFeatures:
    - .NET Framework 4.8 Advanced Services

**Comment by mdanish-kh** [comment, marked as answer]
Dependencies are not yet fully supported by WinGet.
- Blocked by https://github.com/microsoft/winget-cli/issues/685, https://github.com/microsoft/winget-cli/issues/556
